pingen/models/base_config_settings.py

```diff
@@ -27,3 +27,8 @@ class ResConfigSettings(models.TransientModel):
     pingen_staging = fields.Boolean(
         string="Pingen Staging", related="company_id.pingen_staging", readonly=False
     )
+    pingen_ssl_verify = fields.Boolean(
```

Comment:
Where is this field used apart in the view ?

Reply:
Good catch :+1: 
I forgot to add it in `res_company.py`, it is done now